packages/app/src/components/SearchBar/index.js

```diff
@@ -1,23 +1,28 @@
-import React, { useState, useContext, useEffect } from 'react';
+import React, { useState, useContext } from 'react';
 import { Redirect, Link } from 'react-router-dom';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import SettingsModal from 'react-modal';
 import Settings from '../Settings/Settings';
 import GrantCycleNomsResults from '../Settings/GrantCycleNomsResults';
 import { NominationsDataContext } from '../../utils/context/NominationsContext';
+import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
```

Comment:
Are we using this somewhere? I didn't see any other code added.

Reply:
fixed